scripts/sort_contibutors/main.py

```diff
@@ -0,0 +1,129 @@
+import requests
+import json
+import os
+from dotenv import load_dotenv
+
+# It's just for debugging purpose
```

Comment:
What is :) ?

Reply:
*load_dotenv* finds the .env file on current directory where the script is placed.
*os.getenv('USER_TOKEN')* loads the value on variable *USER_TOKEN* .